test-suite/src/test/scala/org/scalajs/testsuite/javalib/util/concurrent/locks/ReentrantLockTest.scala

```diff
@@ -0,0 +1,30 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite.javalib.util.concurrent.locks
+
+import scala.scalajs.js
+
+import org.scalajs.jasminetest.JasmineTest
+
+object ReentrantLockTest extends JasmineTest {
+
+  describe("java.util.concurrent.locks.ReentrantLock") {
+
+    it("should have all the normal operations") {
```

Comment:
Create one `it` per method. (Well you will probably need to use `lock()/unlock()` in most `it`s, but otherwise each `it` should test one method.